src/pas/plugins/oidc/browser/view.py

```diff
@@ -101,7 +102,10 @@ def __call__(self):
         if came_from:
             session.set("came_from", came_from)
 
-        client = self.context.get_oauth2_client()
+        try:
+            client = self.context.get_oauth2_client()
+        except OAuth2ConnectionException:
+            return ""
```

Comment:
@erral in #18 your proposal was '...direct the user to an error page', but here I see an empty return. I assume that for the user it is either an empty page or a 204 http status. This may not be clear to the user. Or am I missing something?

Reply:
indeed

Reply:
Fixed, with the latest changes I redirect the user to the came_from URL if available, and if not to the root of the portal.